etc/chromeos/lib/installs.sh

```diff
@@ -97,6 +139,7 @@ install_utilities() {
 install_hashicorp_tools() {
     echo "Installing hashicorp tools: consul nomad packer"
     curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
+    sudo apt-get install -y lsb-release software-properties-common # for `lsb_release` and `apt-add-repository``
```

Comment:
This probably makes more sense in the install_build_tooling function

Reply:
apt-get is deprecated and apt-key is too

You can do this instead like so:
```
# Let's get Consul and Nomad installed
RUN curl -s https://apt.releases.hashicorp.com/gpg | \
    gpg --no-default-keyring --keyring gnupg-ring:/etc/apt/trusted.gpg.d/hashicorp-apt.gpg --import && \
    chmod 644 /etc/apt/trusted.gpg.d/hashicorp-apt.gpg
```